src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -271,6 +272,12 @@
         + "renaming map produced should be saved")
     private String variableMapOutputFile = "";
 
+    @Option(name = "--instrument_param_mapping_report",
```

Comment:
Nit: just `--instrumentation_mapping_report` is cleaner. `param` is pretty generic and doesn't really add value.